pkg/distro/defs/fedora/distro.yaml

```diff
@@ -1884,46 +1852,69 @@ image_types:
       <<: *iot_simplified_installer_partition_tables
     package_sets:
       installer:
-        - *installer_pkgset
         - include:
+            - "anaconda-dracut"
+            - "atheros-firmware"
             - "attr"
             - "basesystem"
             - "binutils"
+            - "brcmfmac-firmware"
             - "bsdtar"
             - "clevis-dracut"
             - "clevis-luks"
             - "cloud-utils-growpart"
             - "coreos-installer"
             - "coreos-installer-dracut"
             - "coreutils"
+            - "curl"
             - "device-mapper-multipath"
             - "dosfstools"
+            - "dracut-config-generic"
             - "dracut-live"
+            - "dracut-network"
             - "e2fsprogs"
             - "fcoe-utils"
             - "fdo-init"
             - "fedora-logos"
             - "gdisk"
             - "gzip"
+            - "hostname"
             - "ima-evm-utils"
             - "iproute"
             - "iptables"
             - "iputils"
             - "iscsi-initiator-utils"
+            - "iwlwifi-dvm-firmware"
+            - "iwlwifi-mvm-firmware"
+            - "kernel"
             - "keyutils"
+            - "less"
+            - "linux-firmware"
             - "lldpad"
             - "lvm2"
             - "mdadm"
+            - "nfs-utils"
             - "nss-softokn"
+            - "openssh-clients"
+            - "ostree"
+            - "plymouth"
             - "policycoreutils"
             - "policycoreutils-python-utils"
             - "procps-ng"
+            - "realtek-firmware"
+            - "rng-tools"
             - "rootfiles"
+            - "rpcbind"
+            - "selinux-policy-targeted"
             - "setools-console"
+            - "shadow-utils"  # includes passwd
             - "sudo"
+            - "systemd"
+            - "tar"
             - "traceroute"
             - "util-linux"
-            - "shadow-utils"  # includes passwd
+            - "xfsprogs"
+            - "xz"
           conditions:
```

Comment:
Not sure how this hunk is related to the PR title, but okay.

Reply:
No hard feelings and I usually do not care as long as commits are separate, I was just curious. Feel free to keep it in!